src/test/routes/userProfiles.spec.ts

```diff
@@ -97,23 +184,54 @@ describe("User profiles page", () => {
           expect(res.text).to.contain("Jurisdiction 3");
         });
     });
+
     it("should return error from the server", () => {
       idamServiceMock.resolveRetrieveUserFor("1", CCD_IMPORT_ROLE);
       idamServiceMock.resolveRetrieveServiceToken();
+      mock("http://localhost:4451")
+        .get("/api/idam/adminweb/authorization")
+        .reply(200, { canManageUserProfile: true });
+
       mock("http://localhost:4453")
         .get("/users")
         .query({ jurisdiction: "Mike" })
         .replyWithError({ code: 500, text: "Server Error" });
 
-      return request(appTest)
+      return request(appTestWithAuthroziedAdminWebRoles)
         .post("/userprofiles")
         .set("Cookie", "accessToken=ey123.ey456")
         .send({
           jurisdictionName: "Mike",
         })
         .then((res) => {
+          // I have not called /users
           expect(res.status).to.equal(500);
         });
     });
+
+    it("should not call /users if user is not authorized", () => {
+      idamServiceMock.resolveRetrieveUserFor("1", CCD_IMPORT_ROLE);
+      idamServiceMock.resolveRetrieveServiceToken();
+      mock("http://localhost:4451")
+        .get("/api/idam/adminweb/authorization")
+        .reply(200, { canManageUserProfile: true });
+
+      mock("http://localhost:4453")
+        .get("/users")
+        .query({ jurisdiction: "Mike" })
+        .replyWithError({ code: 500, text: "Server Error" });
+
+      return request(appTest)
+        .post("/userprofiles")
+        .set("Cookie", "accessToken=ey123.ey456")
+        .send({
+          jurisdictionName: "Mike",
+        })
+        .then((res) => {
+          // I have not called /users
+          expect(res.status).to.equal(200);
```

Comment:
Bit confused here... in an earlier test, not calling `/users` would expect to return a status of `500`, but in this test it's `200`?